### DISCUSSION TITLE
...art/nvim-lsp-installer/lua/nvim-lsp-installer/server.lua:74: attempt to call method 'setup_lsp' (a nil value)打开总是报这种错？

### DISCUSSION THREAD

**Discussion by wxzvim** [nshen/learn-neovim-lua#79] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/110797126/183801516-2f502798-fef5-4b6b-a404-00821d7353f0.png)
jiantu
见上图，vim打开任何文件总是报上面的错，请问咋解决？

**Comment by nshen** [comment]
好像没有找到啥东西，是不是配置没有return呀，参考
https://github.com/nshen/learn-neovim-lua/blob/main/lua/lsp/config/pyright.lua

**Reply by wxzvim** [reply]
![image](https://user-images.githubusercontent.com/110797126/183827657-bfc4ea22-95fb-4eee-af13-5db190faed5c.png)
wojiushi我就是完全抄的你的这个配置，一摸一样，，不知道咋引入这个模块就报错

**Reply by nshen** [reply]
setup.lua 里边呢？

**Reply by wxzvim** [reply]
![image](https://user-images.githubusercontent.com/110797126/183839204-fde1896e-cbf7-4a1a-9671-0d68e9fec832.png)
也是一样的

**Comment by nshen** [comment, marked as answer]
看群里最终破案是把 nvim-lsp-installer 更新到最新版就可以了
